nixos/modules/services/misc/mbpfan.nix

```diff
@@ -70,13 +73,6 @@ in {
   ];
 
   config = mkIf cfg.enable {
-    services.mbpfan.settings = mkIf cfg.aggressive {
```

Comment:
Might be a good idea to mention in release notes that defaults have changed. Maybe include a justification?

Reply:
Release notes mentioning changes?

Reply:
Done